pkg/build/pipeline.go

```diff
@@ -140,7 +140,7 @@ func monitorPipe(pipe io.ReadCloser) {
 func (p *Pipeline) evalRun(ctx *PipelineContext) error {
 	replacer := replacerFromMap(p.With)
 	fragment := replacer.Replace(p.Runs)
-	script := fmt.Sprintf("#!/bin/sh\nset -e\n%s\nexit 0\n", fragment)
+	script := fmt.Sprintf("#!/bin/sh\nset -e\nunset PATH\nsource /etc/profile\n%s\nexit 0\n", fragment)
```

Comment:
`/etc/profile` is not guaranteed to exist, so we cannot do this.

Reply:
perhaps, $PATH should be set to `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin` initially?
I realize I'm in a somehow unusual setup since my host distro is NixOS, so for instance my $PATH is `/run/wrappers/bin:/home/amine/.nix-profile/bin:/etc/profiles/per-user/amine/bin:/nix/var/nix/profiles/default/bin:/run/current-system/sw/bin` which wouldn't work. But I suspect relying on PATH from the host can bring problems anyway and better set it to what the default is for Alpine ?